apps/cesium.omniverse.dev.kit

```diff
@@ -17,6 +18,15 @@ app.useFabricSceneDelegate = true
 app.usdrt.scene_delegate.enableProxyCubes = false
 app.usdrt.scene_delegate.geometryStreaming.enabled = false
 app.fastShutdown = true
+# Both searchPaths settings must be set in order for material graph to find cesium mdl exports
+renderer.mdl.searchPaths.custom = "${app}/../exts/cesium.omniverse/mdl"
+materialConfig.searchPaths.custom = ["${app}/../exts/cesium.omniverse/mdl"]
+materialConfig.materialGraph.userAllowList = ["cesium.mdl"]
 
 [settings.app.exts]
-folders.'++' = ["${app}/../exts"] # Make extensions from this repo available.
+folders.'++' = [
+    "${app}", # Find other applications in this folder
```

Comment:
Why is this change required? I'm not sure I follow.

Reply:
@weegeekps Sean implemented this as part of changes to make the Material Graph Nodes available inside of the Cesium Dev App (which I can confirm is working)

Based on the other changes in that commit, it looks like he has moved a handful of duplicate settings that were repeated across our other apps into the main cesium.omniverse.dev.kit, which makes sense as our other apps use cesium.ominverse.dev.kit as a dependency.